exporters/otlp/logs/src/main/java/io/opentelemetry/exporter/otlp/logs/OtlpGrpcLogExporterBuilder.java

```diff
@@ -100,9 +92,7 @@ public OtlpGrpcLogExporterBuilder setCompression(String compressionMethod) {
     checkArgument(
         compressionMethod.equals("gzip") || compressionMethod.equals("none"),
         "Unsupported compression method. Supported compression methods include: gzip, none.");
```

Comment:
Isn't all the validation in here unnecessary now? Duplicates logic in the delegate.

Reply:
Same feedback applies to the other builders.

Reply:
Ah yeah did want to confirm what people think on this. There is a school of thought that validation mentions are more user friendly being as close to the user code as possible, even if it means some duplicated code (prioritize UX over duplication). I follow this school :) If it's ok I'd like to remove from the delegate and keep in the callers, what do you think?